cmd/logger.go

```diff
@@ -137,7 +92,78 @@ func getSource() string {
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
 
-func logIf(level logrus.Level, source string, err error, msg string, data ...interface{}) {
+func printfLevel(level, msg, time, cause,
```

Comment:
1. Done.
2. `printfLevel` is only called by `logIf`, so it can be incorporated into `logIf`function. This way we only have logIf` function.
Although this makes `logIf` function, originally 31 lines, a little bit longer, 24 more lines, it is still better to have it this way.